src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -30,10 +31,12 @@
 	actions={route !== 'settings'}
 />
 
-<main class="pt-12">
-	<svelte:component this={cmpLoaders}>
-		<slot />
-	</svelte:component>
-</main>
+<div use:pointerEventsHandler>
```

Comment:
In addition it feels also incorrect to wrap main has any pointer events that bubble within the main will be catched. For example clicking on the network selector triggers this even.

Reply:
moved it (see [comment to one of previous](https://github.com/dfinity/oisy-wallet/pull/1796#discussion_r1726381895))